### ISSUE TITLE
Tracheal hemorrhage - PULMO

### ISSUE THREAD

**Issue by katrinknoflach** [obophenotype/human-phenotype-ontology#7077]
**Preferred term label:**
Tracheal hemorrhage

**Synonyms**
Tracheal bleeding

**Definition (free text, please give PubMed ID)**
Tracheal hemorrhage is a focal bleeding within the trachea. It be diagnosed by tracheobronchoscopy.

**Parent term (use hpo.jax.org/app)**
Abnormal tracheal morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

@chrapp, @mgriese

**Comment by pnrobinson** [comment]
added, thank you!
